app/src/forms/file/storage/objectStorageService.js

```diff
@@ -29,6 +29,7 @@ class ObjectStorageService {
       endpoint: this._endpoint,
       accessKeyId: this._accessKeyId,
       secretAccessKey: this._secretAccessKey,
+      signatureVersion:'v4',
```

Comment:
I'm pretty certain our bucket only supports v2 signatures and not v4 signatures.